cypress/integration/examples/local_storage.spec.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
```

Comment:
Would be good to delete ll the example files for production. *SO* interesting, though.

Reply:
Agreed! I hadn't considered this when commiting, but may still be useful to use in future.